bin/pull-image

```diff
@@ -1,9 +1,13 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
-# DOC: Pull the latest development environment docker image from AWS.
+# DOC: Pull the latest development environment docker image from Docker Hub unless USE_LOCAL_CIVIFORM is set.
+
+source bin/lib.sh
 
 if [ -z "$USE_LOCAL_CIVIFORM" ]; then
-  echo "Making sure we're up to date with the latest dev... set environment variable USE_LOCAL_CIVIFORM=1 to skip"
+  echo "Making sure we're up to date with the latest dev... "\
+    "set environment variable USE_LOCAL_CIVIFORM=1 to skip"
```

Comment:
Maybe this is just shell user error, but I only just realized you have to "export USE_LOCAL_CIVIFORM=1" this for it to work...  how about updating the doc?

Reply:
there's a couple ways to set an environment variable, using export is one of them. maybe just as common is to assign a variable in the same statement as the command, which adds it to the shell environment for that command e.g. `USE_LOCAL_CIVIFORM=1 bin/pull-image`

Reply:
Just discussing now:  That's fair, just feels super error prone given how obscure it is for the situation someone would us it; maybe I'll put in the docs, I def got bit by it trying to work on the train a few weeks ago when I was less familiar with things.

I even recalled you could use a semicolon but that's not true or anymore :D.     USE_LOCAL_CIVIFORM=1; bin/pull-image

Reply:
> I even recalled you could use a semicolon but that's not true or anymore :D. USE_LOCAL_CIVIFORM=1; bin/pull-image
yeah the semi-colon separates the variable assignment and command into two different statements so the variable is just a plain variable rather than part of the environment. try running:

`FOO=1 env`
and
`FOO=1; env`